ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,153 @@
+Launch a Virtual Server Instance on IBM Cloud
+=============================================
+
+This tutorial guides you through the process of launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu. Ensure you have an active IBM Cloud account before starting.
+
+Via Web Console
+---------------
+
+#. First, on the IBM Cloud Web Console, navigate to the Virtual Server creation form. To do this, navigate to     
+   :guilabel:`VPC Infrastructure` > :guilabel:`Compute`> :guilabel:`Virtual server instances`>, 
+   and then click on the blue :guilabel:`Create +`.  
+#. Enter a unique name for your VSI in the `Name` field.
+#. Under the image section, press on :guilabel:`Change image` and under the :guilabel:`Stock Images` tab, search for
+   "Ubuntu" in the search bar and select the Ubuntu OS you'd like to use. For the most recent OS, select :code:`ibm-ubuntu-24-04-minimal-amd64-1`
+#. Choose the desired profile for your VSI under the `Profile` section.
+#. Select an existing ssh key or create a new one by pressing on :guilabel:`Add SSH key` and following the instructions.
+#. Customize any other settings as needed.
+#. Press on the blue :guilabel:`Create virtual server` button to launch your VSI.
+
+Via IBM Cloud CLI
+-----------------
+
+**Prerequisites**
```

Comment:
Instead of keeping this as a separate section, just include the prerequisites in a single direct statement here. That way the steps that follow can just be a sequence of headings (that will automatically show up on the in-page navigation at the right side of the page)